src/functions-and-arrays.js

```diff
@@ -29,9 +64,32 @@ const wordsUnique = [
   'bring'
 ];
 
+function  uniquifyArray(wordsUnique) {
+  let newArr = [];
+  wordsUnique.forEach(function(i) {
+    if(!newArr[i]) {
+      newArr[i] = true
+    }
+  })
+  return Object.keys(x)
```

Comment:
I think you want to get the keys from `newArr` and not `x` here.

```suggestion
  return Object.keys(newArr)
```